docs/start/web-servers.md

```diff
@@ -54,7 +54,7 @@ server {
     root /path/to/public;
 
     location / {
-        try_files $uri $uri/ /index.php$is_args$args;
+        try_files $uri /index.php$is_args$args;
```

Comment:
I wonder if the intention was for this to be:

    try_files $uri $uri/index.php$is_args$args;

?